values.schema.json

```
@@ -484,7 +484,7 @@
                                     "type": "boolean"
                                 },
                                 "maxUnavailable": {
-                                    "type": "null"
+                                    "type": "integer"
```

I think this needs to also allow for a type of 'null' since that's the default in values.yaml.

```suggestion
                                    "type": [
                                        "null",
                                        "integer"
                                    ]
```